Antidote/ActiveCallViewController.m

```diff
@@ -181,39 +185,39 @@ - (void)installConstraints
         make.height.equalTo(kEndCallButtonHeight);
     }];
 
-    [self.videoButton makeConstraints:^(MASConstraintMaker *make) {
-        make.bottom.equalTo(self.view.centerY).with.offset(-kIndent);
+    [self.controlsContainerView makeConstraints:^(MASConstraintMaker *make) {
+        make.centerY.equalTo(self.view);
         make.centerX.equalTo(self.view);
-        make.width.equalTo(kButtonSide);
-        make.height.equalTo(kButtonSide);
+        make.height.equalTo(kButtonSide * 3 + 2 * k3ButtonGap);
```

Comment:
You can resize `controlsContainerView` with it's subviews. If you'll set constraints right, it will be resized automatically.

```
------------------- controlsContainerView
|      top        |
|       |         |
|      top        |
|  [ button ]     |
|     bottom      |
|       |         |
|      top        |
|  [ button ]     |
|     bottom      |
|       |         |
|     bottom      |
-------------------
```